.changelog/3988.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: Added retry mechanism for create docker container in lint command.
+  type: internal
```

since it will also run in customer envs, I think it should be a feature
```suggestion
  type: feature
```